### DISCUSSION TITLE
More 3D sceneries?

### DISCUSSION THREAD

**Discussion by Atque** [Stellarium/stellarium#1508] [Q&A] [answered]
Stellarium only comes with two sceneries, out of which only one (Sterngarten) is fun to use. Is there any plans of extending the number of sceneries, and is there a place (such as the Stellarium start page) where more sceneries can be downloaded?

**Comment by gzotti** [comment, marked as answer]
Good question. If you make a scenery of a real-world landscape and release it under a CC license, we can host it for download. My hope in 2016 was to build up a collection of culturally important buildings with astronomical relations provided by a multitude of users, just like the landscapes. So far it has turned out that heritage institutions don't encourage distributing their precious data, which is pretty frustrating. A few models may come within months, but currently I am too busy for real progress.  *Be the first to contribute!*
